.github/workflows/test.yml

```
@@ -0,0 +1,66 @@
+name: test
```

💅 Suggest to update the name:

```suggestion
name: Run tests
```